project/Build.scala

```diff
@@ -114,7 +114,7 @@ object Build {
       )
 
   // Compiler plugins
-  lazy val nscPlugin = MultiScalaProject("nscplugin", file("nscplugin"))
+  lazy val nscPlugin = MultiScalaProject("nscplugin", file("nscplugin"), "2.13")
```

Comment:
Can we use a named argument here please? Otherwise it's really hard to reason what it means. Also probably a list instead of varargs would be easier to maintaine here, especially if we would add another parameters and since most of other parameters are Strings as well

Reply:
Sure! done https://github.com/scala-native/scala-native/pull/3450/commits/fedb4f35c1292863175ff27372a0886e3ae02b6c